docs/config-overview.md

```diff
@@ -213,3 +215,31 @@ and has the following schema:
 | name             |      type      | description |
 |------------------|----------------|-------------|
 | source           | string         | The file path to be imported |
+
+
+# vendor block schema
+
+The `vendor` block has no labels, **do not** support [merging](#config-merging)
+and has the following schema:
+
+| name             |      type      | description |
+|------------------|----------------|-------------|
+| [manifest](#vendormanifest--block-schema) | block | The manifest for which files to vendor |
+
+# vendor.manifest block schema
+
+The `vendor.manifest` block has no labels, **do not** support [merging](#config-merging)
+and has the following schema:
+
+| name             |      type      | description |
+|------------------|----------------|-------------|
+| [default](#vendormanifestdefault--block-schema) | block | The default manifest |
+
+# vendor.manifest.default block schema
+
+The `vendor.manifest.default` block has no labels, **do not** support [merging](#config-merging)
+and has the following schema:
+
+| name             |      type      | description |
+|------------------|----------------|-------------|
+| files            | list(string)   | The list of [gitignore patterns](https://git-scm.com/docs/gitignore) to match selected files |
```

Comment:
```suggestion
| files            | list(string)   | The list of patterns to match selected files. The pattern format is the same of [gitignore](https://git-scm.com/docs/gitignore#_pattern_format) |
```

I'm not sure if this improves but the other way it looks like the option is related to git, when it's not, we only use gitignore pattern format.. feel free to ignore.